### DISCUSSION TITLE
Different behaviors for `closed` pull_request event

### DISCUSSION THREAD

**Discussion by jlefebvre1997** [community/community#36664] [Actions]
Hello there, 

I don't know if this is the best place to ask, feel free to redirect me if this isn't ! 

As stated in the title, different behaviors occur when using the `pull_request` `closed` event : 
- If manually closing the pull request, the `$GITHUB_REF` variable value is `refs/pull/:prNumber/merge`, as stated in the documentation.
- If merging the branch (which will close the associated pull request), the `$GITHUB_REF` variable value is the name of the branch which receives the changes (which isn't stated anywhere in the doc, iirc) 

I should precise that the goal of the workflow is to trigger a CircleCI job, with https://github.com/CircleCI-Public/trigger-circleci-pipeline-action, which uses the `$GITHUB_REF` variable in case it doesn't start with `refs/pull`, else it will use `$GITHUB_HEAD_REF`, which does contain the name of the base branch. 

Is it intended ? If yes, is there a way to bypass this and have `$GITHUB_REF` actually contain the base branch when merging a pull request ? 

Thanks in advance for you help !

**Comment by jlefebvre1997** [comment]
:up:

**Comment by ossanna16** [comment]
Hey there @jlefebvre1997 and thank you for asking a good question! I'm glad you found our community 🙂
